src/app/cup/cup-rating-position-in-groups/cup-rating-position-in-groups.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CupRatingGroup } from '@models/v2/cup/cup-rating-group.model';
+import { CupRatingService } from '@app/core/services/api/v2/cup/cup-rating.service';
+import { PaginatedResponse } from '@models/paginated-response.model';
+import { CupRatingPositionSearch } from '@app/shared/models/search/cup/cup-rating-position-search.model';
+
+@Component({
+   selector: 'app-cup-rating-position-in-groups',
+   templateUrl: './cup-rating-position-in-groups.component.html',
+   styleUrls: ['./cup-rating-position-in-groups.component.scss']
+})
+export class CupRatingPositionInGroupsComponent implements OnInit {
+   public competitionId!: number;
+   public position!: number;
+   public ratingData!: CupRatingGroup[];
+   public furtheranceData!: number[];
+
+   constructor(private route: ActivatedRoute, private cupRatingService: CupRatingService) {}
+
+   ngOnInit(): void {
+      this.route.paramMap.subscribe(params => {
```

Comment:
It's not necessary to subscribe on the params change here. There is no posibility to navigate for example from `/cup/194/rating-position-in-groups/3` to `/cup/194/rating-position-in-groups/2`. So you can do `this.activatedRoute.params.forEach`. [Example](https://github.com/andrewshostak/prognoz_web_app/blob/master/src/app/championship/championship-match/championship-match.component.ts#L37)